tests/devices.spec.ts

```diff
@@ -106,3 +106,61 @@ test("profile edit networks", async ({ page }) => {
 
   await deleteProfile(page, profile);
 });
+
+test("add, edit and remove a proxy device", async ({ page }) => {
+  const profile = randomProfileName();
+
+  //Add Proxy Device
+  await startProfileCreation(page, profile);
+  await page.getByText("Proxy", { exact: true }).click();
+  await page.getByRole("button", { name: "New Proxy Device" }).click();
+  await page.getByLabel("Bind").selectOption({ index: 2 });
+  await page.getByPlaceholder("127.0.0.1").first().fill("127.0.0.1");
+  await page.getByPlaceholder("[-00]").first().fill("3000");
+  await page
+    .locator('[id="devices\\.0\\.connect\\.address"]')
+    .fill("127.0.0.2");
+  await page.getByPlaceholder("[-00]").nth(1).fill("3001");
+  await finishProfileCreation(page, profile);
```

Comment:
I think we should assert the values from creation as well. We already assert the ones after create, then edit below. But the values directly after creation could be missing or wrong.